### DISCUSSION TITLE
How do you plot the equation curve

### DISCUSSION THREAD

**Discussion by xxtemps** [ansys/pyansys-geometry#1060] [Q&A]
Can Pyansys geometry draw equation curves? Can we rotate the drawn sketch into a solid? I tried plotting the equations in spaceclaim, but some operators didn't work, like ^, and didn't seem to work as a python script.

**Comment by RobPasMue** [comment]
Hi @xxtemps - the sketching API on PyAnsys Geometry is completely client-side meaning that it only "lives" in PyAnsys Geometry and not on your CAD backend (in this case, SpaceClaim). You can leverage PyVista if you want to draw equation curves but they will not be available on SpaceClaim

Revolving a sketch is an upcoming feature that the team is working on, together with being able to sweep a profile- See #1056 for more information.

What is it you want to plot using equations? Would there be a way to discretize it using the ``Segment`` API in Python directly?
